src/main/java/service/read/impl/ProcessFileContentImpl.java

```diff
@@ -0,0 +1,50 @@
+package service.read.impl;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import model.FruitTransaction;
+import service.read.OperationTypeService;
+import service.read.ProcessFileContent;
+
+public class ProcessFileContentImpl implements ProcessFileContent {
+    private static final String CSV_SEPARATOR = ",";
+    private static final String OPERATION_TYPE_NAME = "type";
+    private static final String FRUIT_QUANTITY_NAME = "quantity";
+    private static final String FRUIT_NAME = "fruit";
+    private static final int TYPES_INDEX = 0;
+    private static final int TYPE_INDEX = 0;
+    private static final int FRUIT_NAME_INDEX = 1;
+    private static final int FRUIT_QUANTITY_INDEX = 2;
+
+    private final OperationTypeService operationTypeService = new OperationTypeServiceImpl();
+
+    @Override
+    public List<FruitTransaction> processFile(List<String> fileContent) {
+        String[] columnNames = fileContent.get(TYPES_INDEX).split(CSV_SEPARATOR);
+        int[] typeIndexes = getTypesIndexes(columnNames);
+        List<String> transactions = fileContent;
+        transactions.remove(TYPES_INDEX);
+        return transactions.stream()
```

Comment:
You can use transactions.stream().skip(1) to skip first line with colum names. By this way you don't need remove and copy list (line 25,26)